modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-data-money/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+const grantNumber = Number(
+	document.querySelector('.grant-money div h2').innerText
+);
+
+document.querySelector('.grant-money div h2').innerHTML = Liferay.Util.escape(
+	new Intl.NumberFormat('en-GB', {
+		compactDisplay: 'short',
+		notation: 'compact',
+	}).format(grantNumber)
```

Comment:
```suggestion
	}).format(grantMoney)
```

Reply:
If you put the variable name clear to others to read, it's not necessary to specify the type of the data.